js/classes/entities/foreground/dialog_bubble.js

```diff
@@ -1,147 +1,207 @@
 /**
- * The section of dialog that is displayed near the speaker.
- * @author Nathan Hinthorne
+ * A DialogBubble is a unit of an entity's conversation array. It contains a portion of 
+ * text, which is being said by a specified speaker (or not, as in the case of gameplay instruction).
+ * It <will be> displayed at the bottom of the screen, and show the Speaker's head animation above it.
+ * 
+ * THIS CLASS IS NOT COMPLETE!
+ * 
+ * @author Nathan Hinthorne, Devin Peevy
```

Comment:
I think multiple authors should be put on separate lines each with their own @author tag, e.g.
```
@author Nathan Hinthorne
@author Devin Peevy
```

Reply:
Really? That's dope and makes sense, thank you!